modules/shortcodes/inline-pdfs.php

```diff
@@ -21,10 +21,18 @@ function jetpack_inline_pdf_embed_handler( $matches, $attr, $url ) {
 	/** This action is documented in modules/widgets/social-media-icons.php */
 	do_action( 'jetpack_bump_stats_extras', 'embeds', 'inline-pdf' );
 
-	return sprintf(
-		'<object data="%1$s" type="application/pdf" width="100%%" height="800">
-		  <p><a href="%1$s">%1$s</a></p>
-		</object>',
-		esc_attr( $url )
-	);
+	if ( class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ) {
+		return sprintf(
+			'<p><a href="%1$s">%2$s</a></p>',
+			esc_url( $url ),
+			esc_html__( 'PDF Document', 'jetpack' )
+		);
+	} else {
+		return sprintf(
+			'<object data="%1$s" type="application/pdf" width="100%%" height="800">
+				<p><a href="%1$s">%1$s</a></p>
+			</object>',
+			esc_attr( $url )
+		);
+	}
```

Comment:
It seems that the else case is not needed here since both blocks returns

```suggestion
	}
	return sprintf(
		'<object data="%1$s" type="application/pdf" width="100%%" height="800">
			<p><a href="%1$s">%1$s</a></p>
		</object>',
		esc_attr( $url )
	);
```

Reply:
Good call! Change made, so this is going to need another review :)